Trackovid19-web/src/manifest.webmanifest

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Covidografia",
+    "short_name": "Covidografia",
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "display": "standalone",
+    "scope": "./",
+    "start_url": "./",
```

Comment:
This URL can be updated with a query parameter for analytics porpuses. Something like:
`"start_url": "./index.html?source=a2hs",`.
Setting the `source` query parameter to `a2hs` (add to home screen) is a good PWA practice.

Reply:
👍